docs/custom-pipelines.rst

```diff
@@ -249,5 +251,54 @@ the entry point to the pipeline under the ``[options.entry_points]`` section.
         pipeline_name = scancodeio_scan_and_report_pipeline.pipelines.scan_and_report:ScanAndReport
 
 .. tip::
-    Take a look at `Google LicenseClassifier pipeline for Scancode.io <https://github.com/nexB/scancode.io-pipeline-glc_scan>`_
+    Take a look at `Google License Classifier pipeline for Scancode.io <https://github.com/nexB/scancode.io-pipeline-glc_scan>`_
     for a complete example on packaging a custom tool as a pipeline.
+
+Publishing Pipeline to PyPI
+---------------------------
+After successfully packaging a pipeline, one may want to distribute it as a plugin via PyPI.
```

Comment:
After successfully packaging a pipeline, you may consider distributing it—as a plugin—via PyPI.
Ensure a directory structure similar to the :ref:`pipeline_packaging_example` with all
package files correctly configured.